### PR TITLE
Enclosures

### PR DESCRIPTION
**Pull request by Andriykooo**
- [DEMO LINK](https://Andriykooo.github.io/layout_enclosures/) <br>
- [TEST REPORT LINK](https://Andriykooo.github.io/layout_enclosures/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+body {
+  background-color: #000;
+  margin: 0;
+}
+
+.upper-block,
+.middle-block,
+.inner-block {
+  border: 1px solid #e95d49;
+  box-sizing: border-box;
```
Comment (vsm2k17) [inline src/style.css:10]: add box-sizing on * selector 
 ```
* {
  box-sizing: border-box;
}

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.upper-block,
+.middle-block,
+.inner-block,
+p {
+  margin: 10px;
+}
```
Comment (vsm2k17) [inline src/style.css:18]:
```suggestion
.middle-block,
.inner-block {
  margin: 10px;
}
```

File: src/index.html [hunk 3]
```diff
@@ -1,14 +1,41 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="upper-block" id="qa-block">
+      <div class="middle-block">
+        <div class="inner-block">
+          <p>
+            In programming languages, a closure is a technique for implementing lexically scoped name binding in a language with first-class functions. Operationally, a closure is a record storing a function together with an environment.
```
Comment (vsm2k17) [inline src/index.html:16]: 80 symbols max per line

**Review by mykola-yarchuk — CHANGES_REQUESTED** [review]
good job, just a couple fixes

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+.middle-block,
+.inner-block {
+  border: 1px solid #e95d49;
+}
+
+.middle-block,
+.inner-block,
+p {
+  margin: 10px;
+}
```
Comment (VinnieJ-2k20) [inline src/style.css:20]: Don't style `p` tag here, use classes

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
One small fix and we're OK

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**